docs/learn/dapp-staking-v3/dapp-staking-faq.md

```diff
@@ -0,0 +1,82 @@
+
+### Practical explanation with an example:
+
+Locking tokens means they become eligible to be used for dApp staking. This is similar (or same) as before in v2. Locked tokens cannot be used to pay transaction fees, cannot be sent to other accounts, etc.
+
+In v2, both _locking_ & _staking_ were a single action (_stake_) but in v3 they are split. Once v3 is deployed, all stakes (TVL) will be migrated to locked state within dApp Staking.
+
+This means if `Alice` was _staking_ on dApp `DAPP_1` 3000 ASTR, after the migration, `Alice` will have 3000 ASTR _locked_. But these 3000 ASTR won’t be _staked_ on anything. Instead, `Alice` will need to decide what to stake on (maybe on dApp `DAPP_1` again) and do the 2nd step - _stake_ action.
+
+In case she doesn’t want to stake anymore, she can _unlock_, which is similar to the old _unbond&unstake_ and wait for the unlocking period to pass (roughly 10 days) before these funds can be freely used again (e.g. transfer to someone else).
+
+No stakes are getting carried over, but all TVL is getting carried over.
+
+### Q: What About Unclaimed Rewards?
+Once v3 is launched, all of the unclaimed v2 rewards will become inaccessible. It doesn’t make sense to migrate rewards since the new [Tokenomics 2.0](/docs/learn/tokenomics2/) are tightly coupled with dApp staking - e.g. it’s not really possible to guarantee any inflation rate if we allow minting of over 2 year old rewards.
+
+To prevent users from loosing long accumulated rewards, a special state will be introduced into v2: **Decommission Mode**. Once decommissioning has started, majority of the dApps staking v2 functionality will be disabled, with the exception of reward claim calls. Two new special calls will also become available - _delegated staker reward claim_ & _delegated reward destination setting_.
+
+Using the new delegated calls, anyone will be able to claim rewards for anyone else. E.g. `Alice` can claim rewards for `Bob` - this essentially means that `Alice` pays for the transaction fee to claim `Bob's`. Of course, `Bob's` rewards are deposited into his account, not `Alice's` :slightly_smiling_face:.
+
+Astar team will ensure that all pending rewards are claimed during the decommissioning. We won’t launch v3 until all unclaimed rewards have been claimed.
+
+These transactions won’t be _free_, and whoever runs the _claim bot_ will need to be reimbursed by the treasury - but this is another topic. **TODO - please refer [here](https://forum.astar.network/t/dapp-staking-migration-from-v2-to-v3/5807) for more info.**
+
+Please do your best to claim the rewards yourself, and encourage others to do the same. This is just a one time thing, and won’t be something used again in dApp staking v3. Everyone is responsible for claiming their own rewards - also, with the new periods system, stakers will need to claim rewards & stake again roughly every 3 months to continue earning rewards.
```

Comment:
I'd suggest to trim this down to 2-3 sentences and provide the link to the forum.

It really looks strange that there are smiley's in the official docs.

Reply:
OK.